src/month.rs

```diff
@@ -161,6 +161,22 @@ impl Month {
             Month::December => "December",
         }
     }
+
+    ///     Get the length of the month in a given year
+    pub const fn length(&self, year: i32) -> u8 {
+        match *self {
+            Month::January
+            | Month::March
+            | Month::May
+            | Month::July
+            | Month::August
+            | Month::October
+            | Month::December => 31,
+            Month::April | Month::June | Month::September | Month::November => 30,
+            Month::February if year % 4 == 0 && (year % 25 != 0 || year % 16 == 0) => 29,
```

Comment:
The logic for `February` isn't fully accurate, and I think we have API for this on `Date` that you could repurpose.

Reply:
I found the functions in question under NaiveDate and then under internals.rs but it seems to be meant for testing purposes only and it's unreachable from the month.rs file

Reply:
Feel free to expose whatever this needs as `pub(crate)`, I don't want to duplicate this logic here.